database/postgresQLSeed.js

```diff
@@ -47,17 +50,14 @@ var idToStr = (id) => {
     }
     if (sequentialId % APPEND_CHUNK_LENGTH > APPEND_CHUNK_LENGTH - 6) {
       try {
-        await fs.appendFile(OUTPUT_SQL_FILE_WORKSPACES, workspacesSeedStr);
-        await fs.appendFile(OUTPUT_SQL_FILE_PHOTOS, photosSeedStr);
+        await fs.writeFile(OUTPUT_SQL_FILE_WORKSPACES, workspacesSeedStr);
```

Comment:
So now you are just overwriting the file each time and copying it? That sounds like a great improvement!